src/Routes.js

```diff
@@ -0,0 +1,26 @@
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+  Redirect,
+} from "react-router-dom";
+import Home from "pages/home";
+import Deals from "pages/deals";
+import Cart from "pages/cart";
+const Routes = () => {
+  return (
+    <Router>
+      <Switch>
+        <Route exact path="/home" component={Home} />
```

Comment:
use

```<Route exact path="/">
  <Home />
</Route>
```

Reply:
@ljmerza Previously, I did set the home page path to '/'. However, after that I realised there were some issues with the display of the active state of the nav items when you navigate to Deals or Cart page. The NavLink component from React Router shows Home as active on these pages as well.

Reply:
sorry i didnt notice the /home, i mean use children component instead of props. weird you're having an issue with / though.

Reply:
@ljmerza Ok, thanks for the tip. I'll work on refactoring it such that it uses children components instead of component props.
After some investigation, I found out why the NavLink was matching all the other routes, turns out there is a exact prop I should have used for my Home NavItem so it only shows as active when the path matches exactly '/'.

Reply:
@ljmerza I've pushed and refactored the routes and added exact prop to NavLink component.